test/calliope/render_test.exs

```diff
@@ -4,7 +4,7 @@ defmodule CalliopeRenderTest do
   use Calliope.Render
   import Support.EquivalentHtml
 
-  def haml_with_args, do: "%a{href: '#\{url}'}= title"
+  def haml_with_args(), do: "%a{href: '#\{url}'}= title"
```

Comment:
(Optional) This part can also be written as a module attribute like below because this value is constant and used only in this test.

```elixir
  @haml_with_args "%a{href: '#\{url}'}= title"
```

Reply:
Thank you for your suggestion. It looks great!

However, the purpose of this repository is to support new Elixir versions, not to improve code quality or fix bugs. Therefore, I'd like to keep the original code as much as possible.